Packages/AREQUIPA-IN-LUXURY-PACK/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="atg-location" content="Arequipa">
```

Comment:
not useful meta tag

Reply:
missing all the other good tags (like graph and such): https://gist.github.com/lancejpollard/1978404

Reply:
ok, ill remove all the custom meta tags and put the recommended meta tags